### DISCUSSION TITLE
Support YI Dome Camer U *SFUS?

### DISCUSSION THREAD

**Discussion by gentleman1979** [roleoroleo/yi-hack-Allwinner-v2#720] [Q&A]
I buy this dome camera (Version:9.0.21.13):
![20230930_184956](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/121397274/94d3e34a-bd55-455f-91c2-9eb4f1a52f9c)

Is this model supported?

**Comment by roleoroleo** [comment]
Try h51ga.

**Reply by gentleman1979** [reply]
thx! in works!!!!

**Comment by alexh3o** [comment]
Hi, i can report that my Yi Dome U Full HD received this week, firmware 9.0.22.15, works with h52ga model (sn started with sh52ga so I gave it a try) :)

**Comment by lolorc** [comment]
I also confirm h51ga works on on SFUS, i'm not impressed by video quality tho, probably because of the old firmware

**Comment by roleoroleo** [comment]
Ok, thanks.

**Comment by Ytnom2404** [comment]
Hello , i got a SFUS 9.0.21 YI Dome U 2k version ,....which does not work for me with h51ga files...
The SD card shows factory-done folder etc but nothing ...the yi home app (setup done before) does not show the current camera and the browser also does not open the cam stuff..

Any idea why it does not work for me??

Greetings
Monty

**Reply by Ytnom2404** [reply]
and i can confirm the device is wifi connected

**Reply by roleoroleo** [reply]
Connect to the cam with a telnet session and run these commands:
```
ps
ls -laR /tmp/sd/yi-hack
```

**Reply by Ytnom2404** [reply]
I resolved it… I copied wrong files:-)
